components/public-api/gitpod/v1/workspaces.proto

```diff
@@ -69,6 +72,13 @@ message GetWorkspaceResponse {
     Workspace result = 1;
 }
 
+message GetOwnerTokenRequest {
+    string workspace_id = 1;
+}
+message GetOwnerTokenResponse {
+    string result = 1;
```

Comment:
Normally, `result` would be an object to allow for the returned set to be extended. Here it would look like:
```
message WorkspaceOwnerToken {
  string token = 1;
}
```
Or alternatively, I'd rename `result` to just `token`

Reply:
Renamed in `eab9f306c`